runtime/src/enclave_rpc/dispatcher.rs

```diff
@@ -96,11 +100,20 @@ where
 pub struct Method {
     /// Method dispatcher.
     dispatcher: Box<dyn MethodHandlerDispatch + Send + Sync>,
+    /// Specifies whether the method has to be called over a secure channel.
+    is_secure: bool,
+    /// Specifies whether the method has to be called locally.
+    is_local: bool,
```

Comment:
Should we rather put these fields into the `MethodDescriptor`? This would also make the method registrations more clear as the parameters would be named.